test/forms/BusinessContactDetailsFormProviderSpec.scala

```diff
@@ -82,14 +100,22 @@ class BusinessContactDetailsFormProviderSpec extends StringFieldBehaviours {
     val fieldName = "emailAddress"
     val requiredKey = "businessContactDetails.error.emailAddress.required"
     val lengthKey = "businessContactDetails.error.emailAddress.length"
+    val invalidKey = "businessContactDetails.error.emailAddress.invalid"
+    val validData = "bar@example.com"
     val maxLength = 254
 
     behave like fieldThatBindsValidData(
       form,
       fieldName,
-      stringsWithMaxLength(maxLength)
+      validData
     )
 
+    "must not bind invalid email address data" in {
```

Comment:
Could we add a couple more tests the check the regex IE: emails missing an @ or a .something.

Just for a little more confidence that the regex works as expected